zerver/lib/webhooks/git.py

```diff
@@ -242,6 +242,9 @@ def get_pull_request_event_message(
 
     if message:
         main_message += "\n" + CONTENT_MESSAGE_TEMPLATE.format(message=message)
+
+    if body:
+        main_message += "\n" + CONTENT_MESSAGE_TEMPLATE.format(message=body)
```

Comment:
I think it might just be best if we just pass this event message into `message` rather than creating a dulicate version of it.
Also, I don't think we really need 3 seperate commits for these changes, it's fairly straight forward.

Reply:
Thanks for reviewing my work. You are right the event message can be passed into `message` and that also works perfectly.